scripts/reap_stale_remote_branches.py

```
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import json
+import subprocess
+
+REPO = "organvm/peer-audited--behavioral-blockchain"
+ROOT_DIR = "/Users/4jp/Workspace/peer-audited--behavioral-blockchain"
+
+def run_cmd(cmd):
+    res = subprocess.run(cmd, shell=True, capture_output=True, text=True, cwd=ROOT_DIR)
```

Ruff (0.15.21)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)